### DISCUSSION TITLE
未来会不会考虑出一个tailwindcss版本的antd呢？类似shadcn/ui

### DISCUSSION THREAD

**Discussion by PuddingJoker** [ant-design/ant-design#51817] [General]
未来会不会考虑出一个tailwindcss版本的antd呢？类似shadcn/ui

**Comment by petercat-assistant** [comment]
目前，Ant Design 官方并没有发布关于推出 Tailwind CSS 版本的计划。不过，Ant Design 是一个开源项目，社区开发者可以根据需求自行定制和扩展。如果你有兴趣，可以尝试结合 Tailwind CSS 和 Ant Design 来创建一个自定义版本。

如果你希望官方能够考虑这个方向，可以在 GitHub 上的 Ant Design 仓库中提出一个新功能请求，看看社区和开发团队的反馈。

如需进一步帮助，请在评论中描述您的问题并 @petercat-assistant 与我开始对话。

**Comment by Wxh16144** [comment]
没有（从我在他们贡献群划水这段时间得出的结论）

感觉下一步是 v6，还是继续堆功能。

可以关注一下这个 RFC https://github.com/ant-design/ant-design/discussions/50592 ，或许是另外一个方案
